### PR TITLE
[READY] Walks back combat refactor, plus ammo balance changes

### LINKED ISSUE #8455: [GENERAL] Rubbershot does not always do halloss
#### Brief description of the bug
Rubbershot- which should do halloss to stun people- does not, in some guns. I know for sure it doesn't work in the LMG, but this may be an issue in other weapons.

#### What you expected to happen
Shoot man with LMG rubbershot, man falls but not die

#### What actually happened
Shoot man with LMG rubbershot, man dies

#### Steps to reproduce
- Order an LMG from cargo
- Load with with a medium ammo magazine box (rubbershot)
- Shoot someone and watch them die instead of getting paincritted by halloss

### PR DESCRIPTION
**Pull request by tacoguy7765093**
## About The Pull Request
Reverts most balance changes in #6910 due to balance concerns and trivializing solo-play, the intention is to allow players to work together more against any PvE content, especially now that we got more boons like crawling, removal of mob hitscan, removal of most stuns, easier rescues, surgery buffs, lasers no longer sucking blood, and chemistry buffs we got over the years.

This PR will make projectiles return back to its old damage values making getting hit by bullets feel like getting hit by bullets. This will make solo play and mech-play more risky too as those armor values were never adjusted for the combat update.

Many POIs and mob spawns might need to be redone as an individual is more fatal.

Things like Armor Pen and simplemob bonus damage on phasers have been untouched.

Laser capacity has been reverted(?), so 30 shot lasers are no more.

14.5mm doesn't work properly in the #8171 PR, separating it as a 'super heavy ammo'

Nerfing riot darts from 50 to 20 agony, now printable via autolathe

Rubber bullets no longer bleed you

Fixes #8455 #8732 

Things have been missed and some things are too hyperlethal. Expect armor meta to change in a sperate PR for with these changes.
## Changelog
:cl:
balance: Mob HP has been buffed back to normal values, mostly double the HP
balance: Projectiles are notably more lethal now. Double the amount, damage breakpoints are now 20/40/60 instead of 10/20/40, 
fix: Fixes revolvers using the wrong projectile path.
balance: Laser weaponry now has less shots to compensate for higher damage.
balance: 14.5mm ammo is now separate from big ammo.
balance: Foam and riot foam darts are now printable
balance: increased cost of 14.5mm ammo
fix: Fixed autolathes printing the wrong bullet paths involving 14.5
balance: Foam Riot Darts 50->20
qol: Foam riot darts are now printable
fix: Fixed weapons that fire medium/large bullets firing the wrong projectile.
fix: Fixed rubbershot being as lethal as the real thing.
sound: Added better sounds for the sniper rifles.
balance: Rubbershot doesn't cause bleeding anymore.
/:cl:

### CODE REVIEW

**Comment by TheCaramelion** [comment]
Alright, here we go again.
medic-tf2

**Comment by Skysteak** [comment]
For the sake of just experimentation and for the sake of not presenting as harsh a shock, you may wish to find some arbitrary 'midpoint' or state between the refactor and the old values. From there you can adjust up and down as needed. I wouldn't just rocket back to the old values, although that is more 'gut feeling' than anything else.

**Comment by CMDRYouth** [comment]
This means more Medical Gameplay is on the horizon.
combatmedic_5438

File: code/__defines/simple_mob.dm [hunk 1]
```diff
@@ -5,4 +5,4 @@
 
 // Hivebots are tuned towards how many default lasers are needed to kill them.
 // As such, if laser damage is ever changed, you should change this define.
-#define LASERS_TO_KILL * 15 //chompedit: Reduce health of hivebots by reducing 40 to 15
+#define LASERS_TO_KILL * 40 //chompedit: Reduce health of hivebots by reducing 40 to 15 //CHOMPunedit, back to 40
```
Comment (Kashargul) [inline code/__defines/simple_mob.dm:8]: we don't need chompedits if we revert to upstream code. Remove the entire comment.

File: code/datums/autolathe/arms_ch.dm [hunk 2]
```diff
@@ -71,3 +71,23 @@
 	name = "flamethrower"
 	path =/obj/item/weapon/flamethrower/full
 	hidden = 1
+//CHOMPedit start, these were unticked and I want them in
```
Comment (Kashargul) [inline code/datums/autolathe/arms_ch.dm:74]: No comments in _ch files

**Comment by tacoguy7765093** [comment]
Currently a bug unrelated to this PR needs fixing before merging

**Review by Kashargul — APPROVED** [review]

**Pull request merged**